<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Arvre somos Nozes</string>
  </property>
  <property name="windowIcon">
   <iconset resource="arquivos.qrc">
    <normaloff>:/Arquivos/noz.png</normaloff>:/Arquivos/noz.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButton_Decrescente">
      <property name="text">
       <string>Decrescente</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton_Crescente">
      <property name="text">
       <string>Crescente</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="pushButton_EmOrdem">
      <property name="text">
       <string>em Ordem</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="pushButton_PosOrdem">
      <property name="text">
       <string>pos Ordem</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="10">
     <widget class="QTableWidget" name="tableWidget_MostrarArvore">
      <property name="styleSheet">
       <string notr="true">border-image: url(:/Arquivos/PUC15.png);
background-color: rgb(255, 255, 255);
gridline-color: rgb(185, 185, 185);
border-color: rgb(185, 185, 185);</string>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>250</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>NOME</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>REPETIÇÃO</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="pushButton_PreOrdem">
      <property name="text">
       <string>pre Ordem</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="lineEdit_Buscar">
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLabel" name="label">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>21</height>
       </size>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Buscar :</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>525</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCARREGAR">
    <property name="title">
     <string>&amp;MENU</string>
    </property>
    <addaction name="actionABRIR"/>
    <addaction name="actionLIMPAR"/>
    <addaction name="actionFECHAR"/>
   </widget>
   <addaction name="menuCARREGAR"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionABRIR">
   <property name="icon">
    <iconset resource="arquivos.qrc">
     <normaloff>:/Arquivos/abrir.png</normaloff>:/Arquivos/abrir.png</iconset>
   </property>
   <property name="text">
    <string>&amp;ABRIR</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionLIMPAR">
   <property name="icon">
    <iconset>
     <normalon>:/Arquivos/limpar.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;LIMPAR</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionFECHAR">
   <property name="icon">
    <iconset resource="arquivos.qrc">
     <normaloff>:/Arquivos/fechar.png</normaloff>:/Arquivos/fechar.png</iconset>
   </property>
   <property name="text">
    <string>&amp;FECHAR</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="arquivos.qrc"/>
 </resources>
 <connections/>
</ui>
